pkg/daemon/update.go

```diff
@@ -810,19 +810,6 @@ func (dn *Daemon) updateOS(config *mcfgv1.MachineConfig) error {
 	return nil
 }
 
-// RHEL 7.6 logger (util-linux) doesn't have the --journald flag
-func (dn *Daemon) isLoggingToJournalSupported() bool {
-	if dn.OperatingSystem == machineConfigDaemonOSRHEL {
```

Comment:
One thing I wanted to call out...this conditional looked wrong to me, so I inverted it above.

Reply:
This was actually completely wrong.

Reply:
Can you create a separate PR for this?

Reply:
it was meant to compare against RHCOS, not OSRHEL

Reply:
Done in https://github.com/openshift/machine-config-operator/pull/867

Reply:
Thanks a lot Colin!